client/lib/importer/importer-config.js

```diff
@@ -41,9 +40,7 @@ function getConfig( { siteTitle = '' } = {} ) {
 					supportLink: (
 						<InlineSupportLink
 							supportPostId={ 67084 }
-							supportLink={ localizeUrl(
-								'https://support.wordpress.com/coming-from-self-hosted/'
-							) }
+							supportLink="https://wordpress.com/support/coming-from-self-hosted/"
```

Comment:
This bunch should still have the `localizeUrl()` call I think?

Reply:
`<InlineSupportLink>` [automatically does this](https://github.com/Automattic/wp-calypso/pull/40542/files/798de8ed57005f0b7a9bb60ee5cddc16726fa6ac#diff-67b32f22116b71523d219ab27f50a395R76).